go.mod

```diff
@@ -9,40 +9,75 @@ require (
 	github.com/Azure/go-autorest/autorest/azure/auth v0.4.2
 	github.com/Azure/go-autorest/autorest/to v0.3.0
 	github.com/Azure/go-autorest/autorest/validation v0.2.0 // indirect
-	github.com/aws/aws-sdk-go v1.13.12
+	github.com/aws/aws-sdk-go v1.28.2
```

Comment:
Was this added as a general version upgrade, or did it come with the upgrade to the v1.18 k8s libraries?

Reply:
It came from the k8s version upgrade: https://github.com/kubernetes/kubernetes/blob/v1.18.4/go.mod#L23